sympy/integrals/integrals.py

```diff
@@ -417,6 +418,15 @@ def doit(self, **hints):
         if self.is_zero:
             return S.Zero
 
+        if isinstance(self.function, Sum):
+            for variable in self.variables:
+                if variable in self.function.limits[0]:
+                    break
+                else:
```

Comment:
It looks like `else:` is not necessary.

Reply:
I will have to check this part. Will get back to you regarding this.